bsp/raspberry-pi/raspi3-64/driver/Kconfig

```diff
@@ -3,6 +3,10 @@ config BSP_SUPPORT_FPU
     bool "Using Float"
     default n
 
+config RT_USING_LVGL
```

Comment:
Why add this config? LVGL can be a online package, Just to enable it on packages menuconfig.